libvirt.go

```diff
@@ -108,6 +110,75 @@ const (
 	DomainXMLFlagMigratable
 )
 
+// DomainCreateFlags specifies options when performing a domain creation.
+type DomainCreateFlags uint32
+
+const (
```

Comment:
this is hard, because as you see some const have holes between values, and benefits of iota not very useful.